rest-service/manager_rest/storage/storage_utils.py

```diff
@@ -58,7 +58,12 @@ def create_default_user_tenant_and_roles(admin_username,
         password=encrypt_password(admin_password),
         roles=[admin_role]
     )
-    admin_user.tenants.append(default_tenant)
+    user_tenant = UserTenant(
```

Comment:
Is this the only way to do it? I think you'll agree that the previous way was much more readable. Won't SQLA just take care of it?

Reply:
Since we want to set the tenant and the role in one shot, the association proxy won't be helpul in this case.

I can maybe rename `UserTenant` to `UserTenantAssoc` to make clearer that we are setting an association, not just a tenant. Would that be better?